File: src/test/java/aoc2024/Day02Test.java
package aoc2024;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class Day02Test {

    @Test
    public void aoc_example_part1() {
        assertEquals("2", new Day02().asTest().solvePart1());
    }

    @Test
    public void aoc_example_part2() {
        assertEquals("4", new Day02().asTest().solvePart2());
    }
}

File: src/test/java/aoc2025/Day02Test.kt
package aoc2025

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class Day02Test {

    @Test
    fun aoc_example_part1() {
        assertEquals("1227775554", Day02().asTest().solvePart1())
    }

    @Test
    fun aoc_example_part2() {
        assertEquals("4174379265", Day02().asTest().solvePart2())
    }
}
